client/src/app/components/Modal.tsx

```diff
@@ -35,10 +33,13 @@ class Modal extends Component<ModalProps, ModalState> {
                     onClick={() => this.handleShow()}
                 >
                     {this.props.icon === "dots" && (
-                        <DotsIcon />
-                    )}
+                        <span className="material-icons">
+                            more_horiz
+                        </span>)}
                     {this.props.icon === "info" && (
-                        <InfoIcon />
+                        <span className="material-icons">
```

Comment:
There is a vertical misalignment here 🙏🏼 You can see an example on the identity page
![image](https://user-images.githubusercontent.com/3624944/156154798-88d75099-05b4-472a-8e1d-c2bca8d4b22c.png)

Reply:
![Screenshot from 2022-03-01 12-48-44](https://user-images.githubusercontent.com/72078237/156164756-ea24377c-fae0-4635-af52-72c35cb22447.png)

Resolved